modules/ob-verifying-cluster-wide-proxy.adoc

```diff
@@ -0,0 +1,17 @@
+// This module is included in the following assembly:
+//
+// * configuring/using-builds-in-a-restricted-environment.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ob-verifying-cluster-wide-proxy_{context}"]
+= Verifying cluster-wide proxy
+
```

Comment:
Add an intro sentence

Reply:
Introduction to both the subsections (what a cluster admin can do) is included in the main introduction itself